.github/workflows/go-windows.yml

```diff
@@ -17,7 +17,7 @@ jobs:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        go-version: [1.24.x, 1.25.x]
+        go-version: [1.25.x]
```

Comment:
might want to stick to `stable` and `oldstable` aliases which would reduce churn https://github.com/actions/setup-go#version-aliases